brands/shop/electronics.json

```diff
@@ -105,8 +105,17 @@
     }
   },
   "shop/electronics|Elektra": {
+    "countryCodes": [
+      "gt",
+      "hn",
+      "pa",
+      "pe",
+      "sv"
+    ],
     "tags": {
       "brand": "Elektra",
+      "brand:wikidata": "Q1142753",
+      "brand:wikipedia": "Grupo_Elektra",
```

Comment:
This needs to be formatted like `"es:Grupo Elektra"`
(language prefix, and underscores converted to spaces)